src/main/scala/io/smartcat/twicas/models/LogisticRegressionTweet.scala

```diff
@@ -0,0 +1,51 @@
+package io.smartcat.twicas.models
+
+import io.smartcat.twicas.summary.{ModelSummary, ParameterOptimization}
+import org.apache.spark.ml.classification.{LogisticRegression, LogisticRegressionModel}
+import org.apache.spark.sql.DataFrame
+
+class LogisticRegressionTweet(logisticRegression: LogisticRegressionModel) extends ClassificationModel {
+  val name: String = "Logistic Regression"
+
+  override def classify(df: DataFrame): DataFrame = logisticRegression.transform(df)
+
+  def params: Map[String, Double] = Map(
+    "threshold" -> logisticRegression.getThreshold,
+    "regularization" -> logisticRegression.getRegParam,
+    "elasticNet" -> logisticRegression.getElasticNetParam
+  )
+}
+
+object LogisticRegressionTweet extends Serializable {
+  val featureColumn = "features"
+  val labelColumn = "label"
+  val predictionColumn = "prediction"
+  val probabilityColumn = "probability"
+
+  def makeModels(trainSet: DataFrame, validationSet: DataFrame,
+                 thresholds: List[Double], regParams: List[Double], elasticNet: List[Double]): ParameterOptimization = {
+
+    val parameters = generator(List(thresholds, regParams, elasticNet))
+
+    val models = parameters.map { case (threshold :: regParam :: elastic :: _) =>
+      train(trainSet, threshold, regParam, elastic)
+    }
+
+    ParameterOptimization(models map (ModelSummary.validation(validationSet, _)))
+
+  }
+
+  def train(df: DataFrame, threshold: Double = 0.5, regParam: Double = 0.0, elasticNet: Double = 0.0): LogisticRegressionTweet = {
```

Comment:
This is normal setup for training
Probability above 50% will we classified as 1 (valuable tweet)
regParam and elasticNet = 0.0 ===> no regularization in training